src/tezos/tezosPlugin.js

```diff
@@ -76,8 +85,12 @@ export class TezosPlugin extends CurrencyPlugin {
   async derivePublicKey (walletInfo: EdgeWalletInfo): Promise<Object> {
     const type = walletInfo.type.replace('wallet:', '')
     if (type === 'tezos') {
-      const keypair = eztz.crypto.generateKeys(walletInfo.keys.mnemonic, '')
-      return { publicKey: keypair.pkh, publicKeyEd: keypair.pk }
+      try {
+        const keypair = eztz.crypto.generateKeys(walletInfo.keys.mnemonic, '')
```

Comment:
I don't think we need a try/catch block here either. If something goes wrong, eztz will presumably throw its own error with its own useful message.